src/generated/cloud/lustre/v1/.repo-metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+    "api_id": "lustre.googleapis.com",
+    "api_shortname": "lustre",
+    "client_documentation": "https://docs.rs/google-cloud-lustre-v1/latest",
+    "distribution_name": "google-cloud-rust",
+    "language": "rust",
+    "library_type": "GAPIC_AUTO",
+    "name_pretty": "Google Cloud Managed Lustre API",
+    "release_level": "preview",
```

Comment:
In the future: would we want to generate clients for services that are not GA? Would we want to mark some clients as GA and not others?

Reply:
If we want to programmatically treat `release_level != GA` differently, that is fine with me.

I veto doing anything manual.

> would we want to generate clients for services that are not GA?

I think so, if they are `v[0-9]+`. Customers may be previewing these services using the Rust libraries.

> Would we want to mark some clients as GA and not others?

I think so.